File: ceemsp-frontend/src/app/_models/ComunicadoGeneral.ts
import {Deserializable} from "./Deserializable";

export default class ComunicadoGeneral implements Deserializable {
  uuid: string;
  titulo: string;
  fechaPublicacion: string;
  descripcion: string;

  deserialize(input: any): this {
    Object.assign(this, input);
    return this;
  }

}

File: database/src/main/java/com/pelisat/cesp/ceemsp/database/model/ComunicadoGeneral.java
package com.pelisat.cesp.ceemsp.database.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Entity
@Table(name = "COMUNICADOS_GENERALES")
@Getter
@Setter
public class ComunicadoGeneral extends CommonModel {
    @Column(name = "TITULO", nullable = false)
    private String titulo;

    @Column(name = "DESCRIPCION", nullable = false)
    private String descripcion;

    @Column(name = "FECHA_PUBLICACION", nullable = false)
    private LocalDate fechaPublicacion;
}
